fdbrpc/ReplicationPolicy.cpp

```diff
@@ -476,3 +476,49 @@ TEST_CASE("/ReplicationPolicy/Serialization") {
 	testReplicationPolicy(1);
 	return Void();
 }
+
+// Test calls per second throughput for PolicyAcross.selectReplica.
+void testReplicationPolicyThroughput(int nTests, bool withAlsoServers) {
+	Reference<IReplicationPolicy> policy = Reference<IReplicationPolicy>(new PolicyAcross(3, "zoneid", Reference<IReplicationPolicy>(new PolicyOne())));
+
+	Reference<LocalitySet> logServerSet = Reference<LocalitySet>(new LocalityMap<int>());
+	auto* logServerMap = (LocalityMap<int>*) logServerSet.getPtr();
+	std::vector<std::string> zoneIds = {"a", "b", "a", "b", "c", "d", "e", "f"};
```

Comment:
Have you tried to make zoneIds longer, like UID? The zoneId in real scenarios is never a single character. 
(Although I think this will only affect initialization time negligible and won't affect the result. Never know until we run it. :) )

Reply:
> Have you tried to make zoneIds longer, like UID?

TBH I don't see a reason to change to UID because:

- Existing code leverages string interning for fast comparison which is critical since the comparison is done on may critical code paths that are all performance sensitive.
- String has better expressiveness than UID, since it can be arbitrarily long.

Reply:
Ah, I meant increasing the zoneIds length to what we typically use.

Reply:
Oh okay. The length of `zondId` won't matter because of the string interning.

Reply:
Yes, in the end, these zoneId strings are represented internally by integers.